include/zephyr/bluetooth/att.h

```diff
@@ -91,6 +91,12 @@ size_t bt_eatt_count(struct bt_conn *conn);
 
 #endif /* CONFIG_BT_EATT */
 
+enum bt_att_chan_option {
+	BT_ATT_CHAN_ANY = 0x0,
+	BT_ATT_CHAN_UNENHANCED = BIT(0),
```

Comment:
Is “unenhanced” a term that the spec uses? If not, please use a term from the spec, or if an appropriate one doesn’t exist I’d suggest “legacy”.

Reply:
"Unenhanced" is used in the spec.
> 3.2.11 ATT bearers
Any ATT bearer that is not an Enhanced ATT bearer, using any other L2CAP channel mode, is known as an Unenhanced ATT beare